airflow/www/templates/airflow/dags.html

```diff
@@ -312,7 +312,7 @@ <h2>DAGs</h2>
       d3.json("{{ url_for('Airflow.dag_stats') }}", function(error, json) {
         for(var dag_id in json) {
             states = json[dag_id];
-            g = d3.select('svg#dag-run-' + dag_id)
+            g = d3.select('svg#dag-run-' + dag_id.replace(/\./g, '__dot__'))
```

Comment:
I wonder if the better fix is to change how the JSON is generated to use the `dag.safe_dag_id` accessor instead.

Reply:
(Which does exactly this, but on the server is all.)

Reply:
I considered doing that, but I felt it should be kept separate from the endpoint response since the conversion is only done for the sake of naming a DOM element. I don't think it'd be great to have the endpoint response keyed on a safe_dag_id rather than dag_id because that's unexpected.